backend/local/readme.md

```diff
@@ -0,0 +1,11 @@
+start a local dynamo db
+
+```
+docker run -p 8000:8000 amazon/dynamodb-local
+```
+
+start the backend
+
+```
+ts-node main.ts
```

Comment:
This will only work if the user has installed `ts-node` globally, right? Given that we are already installing `ts-node` as a dev dependency, I would suggest to create a new `package.json` script `startLocalBackend` or something similar, which uses the `ts-node` binary from the `node_modules` installation:
```json
{
  "scripts": {
    "startLocalBackend": "ts-node main.ts"
    }
}
```

Reply:
I added a set of package.json scripts and also integrated them into the regular workflow. As part of this, I also moved their documentation to the main README file.

Now, `npm start` will fire up everything together while you can also just start a backend via `npm start:local-backend` or a frontend via `npm start:frontend`.